pkgs/development/web/publii/default.nix

```diff
@@ -0,0 +1,102 @@
+{ stdenv
+, lib
+, fetchurl
+, autoPatchelfHook
+, makeShellWrapper
+, wrapGAppsHook
+, alsa-lib
+, at-spi2-atk
+, at-spi2-core
+, atk
+, cairo
+, cups
+, dbus
+, expat
+, glib
+, gtk3
+, libsecret
+, mesa
+, nss
+, pango
+, udev
+, xdg-utils
+, xorg
+}:
+
+stdenv.mkDerivation rec {
+  pname = "publii";
+  version = "0.42.1";
+
+  src = fetchurl {
+    url = "https://getpublii.com/download/Publii-${version}.deb";
+    hash = "sha256-GHGXu/z2L4aJG1O1THPIxnRBdPJOIVuQsZP0zhjTZlo=";
+  };
+
+  dontConfigure = true;
+  dontBuild = true;
+  # dontWrapGApps = true;
```

Comment:
```suggestion
```

Reply:
Changed it, can you help me understand why I need it (if it works without) if I use makeWrapper now? I thought we just have the GAppsWrapper and wrap it with a Binary / ShellWrapper depending on the need?

Reply:
It works without disabling the wraps gapps hook itself but then we double wrap which is undesired (Mainly because it is slower and sets argv0 wrong).

Reply:
Thanks for the explanation I appreciate it, it's already fixed so let me know if there is something else I can do to get it merged ✌🏻️😊